doc/nrf/templates/mesh_model_README.rst

```diff
@@ -0,0 +1,210 @@
+.. _bt_mesh_model_template:
+
+Mesh model template
+###################
+
+.. note::
+   Sections with * are optional.
+
+The XYZ Mesh models allow to remotely control a disco light with a mesh device.
+
+There are two XYZ models:
+
+    * XYZ Server
+    * XYZ Client
+
+.. tip::
+   Explain what this model allows you to do within the broader context of the mesh network.
+   Use max two sentences (full sentences, not sentence fragments).
+   You can go into more detail in the next section.
+   List server and client models that are part of this Mesh model.
+
+Server model
+************
+
+The XYZ Server provides information about the disco light.
+The data is split into the following states: location and color.
+
+The XYZ Server adds the following model instances in the composition data:
+
+    * XYZ Server
+    * XYZ Setup Server
+
+The XYZ Server allows observation of the location states, as it only exposes get-messages for the location states.
+
+.. tip::
+   Describe the server model.
+   Start with a general purpose of the server model.
+   Provide basic information about the states, if the server model has any.
+
+   Additionally, list the model instances of the server model.
+   Say what they allow in relation to states and to each other, describe usage criteria and corner cases.
+
+Configuration*
+==============
+
+The following configuration parameters are associated with the XYZ server model:
+
+    * parameter 1
```

Comment:
Should there be a very short description? Like "* :option:`CONFIG` - the maximum size" or something.